packages/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -43,11 +43,18 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
     nextProps: Readonly<IDonutChartProps>,
     prevState: Readonly<IDonutChartState>,
   ): Partial<IDonutChartState> | null {
-    if (nextProps.height && nextProps.height !== prevState._height && nextProps.width !== prevState._width) {
+    let widthState: { _width: number } | undefined;
```

Comment:
what is this change for.

Reply:
> Previous implementation didn't work correctly when only width prop or only height prop is provided. Creating different derived states from width and height props and combining them later fixes the issue

Chart got distorted when only width prop or only height prop was provided